atlasdb-autobatch/src/main/java/com/palantir/atlasdb/autobatch/DisruptorAutobatcher.java

```diff
@@ -85,6 +88,18 @@ private static ThreadFactory createThreadFactory(String safeLoggablePurpose) {
         return result;
     }
 
+    public List<ListenableFuture<R>> applyBatch(int count) {
+        Preconditions.checkState(!closed, "Autobatcher is already shut down");
+        return IntStream.range(0, count).mapToObj($ -> {
+            SettableFuture<R> result = SettableFuture.create();
+            buffer.publishEvent((refresh, sequence) -> {
```

Comment:
is there a 'publishEvents' version of this, or otherwise a Disruptor mechanism of writing multiple events? Also, I don't think this makes sense as a primitive - null arguments are uncommon - rather I expect you'd want to pass a list of arguments (which in our case is a list of nulls).